storm-multilang/dotnet/src/main/dotnet/Dotnet.Storm.Adapter/Program.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+using CommandLine;
+using Dotnet.Storm.Adapter.Components;
+using Dotnet.Storm.Adapter.Logging;
+using log4net;
+using log4net.Appender;
+using log4net.Config;
+using log4net.Core;
+using log4net.Repository.Hierarchy;
+
+namespace Dotnet.Storm.Adapter
+{
+    class Program
+    {
+        private readonly static ILog Logger = LogManager.GetLogger(typeof(Program));
+
+        static void Main(string[] args)
+        {
+            // Parse command line arguments
+            var parser = new Parser(with => with.EnableDashDash = true).ParseArguments<Options>(args);
+
+            string className = null;
+            string assemblyName = null;
+            string arguments = null;
+            LogLevel level = LogLevel.INFO;
+
+            parser.WithParsed(options =>
+            {
+                className = options.Class;
+                assemblyName = options.Assembly;
+                arguments = options.Arguments;
+
+                // by default TryParse will return TRACE level in case of error
+                Enum.TryParse(options.LogLevel.ToUpper(), out level);
+
+                // if user didn't set TRACE level but we TryParse returned TRACE = parsing exception
+                if (!options.LogLevel.ToLower().Equals("trace") && level == LogLevel.TRACE)
+                {
+                    // reset default log level to INFO
+                    level = LogLevel.INFO;
+                }
+            });
+
+            // Configure logging
+            var repository = LogManager.GetRepository(Assembly.GetEntryAssembly());
+
+            if (File.Exists("log4net.config"))
+            {
+                XmlConfigurator.Configure(repository, new FileInfo("log4net.config"));
+            }
+            else
+            {
+                XmlConfigurator.Configure(repository);
+            }
+
+            // setting up log level
+            Level newLevel = StormAppender.GetLogLevel(level);
+
+            // we want to disable storm logger since storm is not accessible yet
+            StormAppender.Disable(repository, newLevel);
+            Logger.Debug("Storm logger disabled.");
+
+            Logger.Debug($"Current working directory: {Environment.CurrentDirectory}.");
+
+            // we don't want any console logger to be enabled since it is bracking storm multilang protocol
+            // it will not prevent the case other appenders write standard output
```

Comment:
it was hard to understand for me what is going on here, have you thought about a comment like the following:
`// we are disabling all console appenders from log4net as they interfere with the multilang protocol`